dopamine/experiment5/experiment5_statstests.jl

```diff
@@ -0,0 +1,225 @@
+# [2]    8     2  2452259.7245  -58602.0911  0.2725  0.0174  7.3842  0.0007
+# ─────────────────────────────────────────────────────────────────────────
+
+# interaction among all vars
+mwb = lm(@formula(meanspeed ~ medium + worm + bacteria), trackstats)
+mwb_int = lm(@formula(meanspeed ~ medium*worm*bacteria), trackstats)
+
+ftest(mwb.model, mwb_int.model)
+# F-test: 2 models fitted on 625 observations
+# ──────────────────────────────────────────────────────────────────────────
+#      DOF  ΔDOF           SSR          ΔSSR      R²     ΔR²      F*   p(>F)
+# ──────────────────────────────────────────────────────────────────────────
+# [1]    6        2510861.8156                0.2551                        
+# [2]   13     7  2375077.8791  -135783.9365  0.2954  0.0403  5.0065  <1e-04
+# ──────────────────────────────────────────────────────────────────────────
+
+lm(@formula(meanspeed ~ worm*bacteria), subset(trackstats, :medium=>ByRow(!=("DA")))) # just buffer
+# meanspeed ~ 1 + worm + bacteria + worm & bacteria
+
+# Coefficients:
+# ───────────────────────────────────────────────────────────────────────────────────────
+#                                Coef.  Std. Error      t  Pr(>|t|)  Lower 95%  Upper 95%
+# ───────────────────────────────────────────────────────────────────────────────────────
+# (Intercept)                 269.204      10.41    25.86    <1e-78   248.721    289.688
+# worm: CB                    -64.1178     15.0256  -4.27    <1e-04   -93.6832   -34.5525
+# worm: MT                    -45.1968     14.2228  -3.18    0.0016   -73.1826   -17.211
+# bacteria: OP50             -100.851      13.4165  -7.52    <1e-12  -127.25     -74.4519
+# worm: CB & bacteria: OP50    70.0027     18.8285   3.72    0.0002    32.9544   107.051
+# worm: MT & bacteria: OP50    68.086      18.4956   3.68    0.0003    31.6927   104.479
+# ───────────────────────────────────────────────────────────────────────────────────────
+
+lm(@formula(meanspeed ~ worm*bacteria), subset(trackstats, :medium=>ByRow(!=("M9")))) # just dopamine
+# meanspeed ~ 1 + worm + bacteria + worm & bacteria
+
+# Coefficients:
+# ───────────────────────────────────────────────────────────────────────────────────────
+#                                Coef.  Std. Error      t  Pr(>|t|)  Lower 95%  Upper 95%
+# ───────────────────────────────────────────────────────────────────────────────────────
+# (Intercept)                 252.881      10.0241  25.23    <1e-75   233.156   272.607
+# worm: CB                    -32.7666     13.4317  -2.44    0.0153   -59.1975   -6.33565
+# worm: MT                    -30.643      13.8936  -2.21    0.0282   -57.9827   -3.30322
+# bacteria: OP50             -100.472      12.1398  -8.28    <1e-14  -124.36    -76.5829
+# worm: CB & bacteria: OP50    20.7443     16.8004   1.23    0.2179   -12.3155   53.8041
+# worm: MT & bacteria: OP50    12.658      17.4657   0.72    0.4692   -21.7109   47.027
```

Comment:
dopamine
MT on and off bacteria vs N2 on and off bacteria
ie. speed diff of MT vs speed diff of N2 
p = 0.4692 --> not significant